src/app/modules/admin-core/services/users-service/users.config.ts

```diff
@@ -2,5 +2,10 @@
 export const UsersConfig = {
     usersForm: '/admin-service/api/v1/user-creation/getForm',
     userroles : '/admin-service/api/v1/platform-user-roles/getProfile',
-    createuser: '/admin-service/api/v1/user-creation/create'
+    createuser: '/admin-service/api/v1/user-creation/create',
+    organisations: '/admin-service/api/v1/organisations/list?pageSize=10&pageNo=1',
+    userslist: '/admin-service/api/v1/organisations/users/0125747659358699520?pageSize=20&pageNo=1'
```

Comment:
page size and number should not be hardcoded here

Reply:
hardcoded in service